src/ripple/rpc/handlers/AccountOffers.cpp

```diff
@@ -24,6 +24,19 @@
 
 namespace ripple {
 
+void appendOfferJson (std::shared_ptr<SLE const> const& offer,
```

Comment:
I kind of miss @JoelKatz 's old SLE::pointer or whatever it was :)  Did we ever consider SLEPointer or something? I vaguely recall valid reasons for removing them, but would that also apply to a long form name?